api/app/src/models/webhook-request.ts

```diff
@@ -0,0 +1,40 @@
+import { DataTypes, Sequelize } from "sequelize";
+import { BaseModel, defaultModelOptions, ModelSetup } from "./_default";
+
+export type WebhookRequestStatus = "processing" | "success" | "failure";
+
+export class WebhookRequest extends BaseModel<WebhookRequest> {
+  static NAME: string = "webhook_request";
+  declare id: string;
+  declare cxId: string;
+  declare payload: unknown;
+  declare status: WebhookRequestStatus;
+  declare statusAt: Date;
+
+  static setup: ModelSetup = (sequelize: Sequelize) => {
+    WebhookRequest.init(
+      {
+        id: {
+          type: DataTypes.UUID,
+          primaryKey: true,
+        },
+        cxId: {
+          type: DataTypes.STRING,
+        },
+        payload: {
+          type: DataTypes.JSONB,
+        },
+        status: {
+          type: DataTypes.STRING,
+        },
+        statusAt: {
+          type: DataTypes.DATE(6),
+        },
```

Comment:
Looks like we only use this during create(), if that's the case, can't we just use the default createdAt column for this?

Reply:
the status can be updated as we try again, that's why I thought about a diff column... as for why not `updatedAt`, in theory they should be the same, as the only thing that should be updatable on a record in this table is the status, maybe we can use that then?

Reply:
removing